Sources/_CFXMLInterface/CMakeLists.txt

```diff
@@ -0,0 +1,35 @@
+##===----------------------------------------------------------------------===##
+##
+## This source file is part of the Swift open source project
+##
+## Copyright (c) 2024 Apple Inc. and the Swift project authors
+## Licensed under Apache License v2.0
+##
+## See LICENSE.txt for license information
+## See CONTRIBUTORS.md for the list of Swift project authors
+##
+## SPDX-License-Identifier: Apache-2.0
+##
+##===----------------------------------------------------------------------===##
+
+add_library(_CFXMLInterface STATIC CFXMLInterface.c)
+
+target_include_directories(_CFXMLInterface
+    PUBLIC
+        include
+    PRIVATE
+        ../CoreFoundation/internalInclude
+        ../CoreFoundation/include
```

Comment:
CoreFoundation lists `CoreFoundation/include` as a public header. Linking against it will pull this in automatically.